test/test_suites/components/uri.cpp

```diff
@@ -2,6 +2,8 @@
 
 #include <malloy/http/uri.hpp>
 
+#include <iostream>
```

Comment:
Is this really necessary? This include was added to `test.hpp`: https://github.com/Tectu/malloy/blob/a20816b1488824bf185c5a77139c3bbec290b703/test/test.hpp#L12